### DISCUSSION TITLE
请问umi4怎么使用keepalive呢

### DISCUSSION THREAD

**Discussion by wudalei** [umijs/umi#10575] [Q&A] [answered]
umi3一直使用的是umi-plugin-keep-alive

**Comment by fz6m** [comment, marked as answer]
umi 4 可以用这个插件：https://github.com/alitajs/alita/blob/master/packages/plugins/src/keepalive.ts

参考文章：https://juejin.cn/post/7109492504424087566

cc @xiaohuoni 大佬啥时候 juejin 写一篇 umi4 keep alive 的文章，上面那个文章比较复杂，希望有一个简洁版的，这个需求过一段时间就有人问 🌹 。

**Reply by xiaohuoni** [reply]
https://alitajs.com/zh-CN/docs/config/config#keepalive
其实我们都是看这个文档。最终这个功能还是要等 umi 内置。
文章需求我记一下

**Reply by xiaohuoni** [reply]
今天找到空闲写这个文章，发现很熟悉，搜索了一下，发现之前就写过 https://juejin.cn/post/7088705614603354142
